### PR TITLE
Fix license cmd

### PR DESCRIPTION
**Pull request by lbudai**
@czanik : FYI

### CODE REVIEW

File: lib/control/control-commands.c [hunk 1]
```diff
@@ -58,6 +58,25 @@ control_register_command(const gchar *command_name, const gchar *description, Co
...
 
+void
+control_replace_command(const gchar *command_name, const gchar *description, CommandFunction function,
+                        gpointer user_data)
+{
+  GList *command_it =  g_list_find_custom(command_list, command_name,
+                                          (GCompareFunc)control_command_start_with_command);
+  if (!command_it)
+    {
+      msg_error("Failed to replace control command",
```
Comment (lbudai) [inline lib/control/control-commands.c]: hmm... I think we could register the command when we want to replace a non-existent (and print a warning msg): I'll change it.

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: lib/control/control-commands.c [hunk 2]
```diff
@@ -58,6 +58,26 @@ control_register_command(const gchar *command_name, const gchar *description, Co
...
+control_replace_command(const gchar *command_name, const gchar *description, CommandFunction function,
+                        gpointer user_data)
+{
+  GList *command_it =  g_list_find_custom(command_list, command_name,
+                                          (GCompareFunc)control_command_start_with_command);
+  if (!command_it)
+    {
+      msg_warning("Trying to replace a non-existent command. Command will be registered as a new command.",
+                  evt_tag_str("command", command_name));
+      control_register_command(command_name, description, function, user_data);
```
Comment (bazsi) [inline lib/control/control-commands.c]: Do we need the warning here? 

If we do, I think we need a bit more context, like mentioning that this was a control socket command, but I like the idea that modules can register such stuff.

But all in all I would make this msg_debug().

**Comment by gaborznagy** [comment]
I think nothing protects from re-registering a command with `control_register_command`, although you cannot override an already registered due to GList find.

We should rethink the `control_register_command` behaviour.

File: lib/control/control-commands.c (hunk 2 above)
Comment (lbudai) [reply lib/control/control-commands.c]: changed to debug

**Comment by lbudai** [comment]
@gaborznagy : fixed

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**